resources/builder.inc.sh

```diff
@@ -1945,12 +1945,13 @@ _builder_has_function_been_called() {
 #   linux      Linux
 #
 # Development tooling platforms:
-#   delphi     (On Windows only, Delphi is installed)
+#   delphi     Delphi is installed (on Windows only)
+#   android    Android Studio is installed
 #
 builder_describe_platform() {
 
   local builder_platforms=(linux mac win)
-  local builder_tools=(delphi)
+  local builder_tools=(android-studio delphi)
```

Comment:
Will there be a similar one for xcode?

Reply:
Perhaps. This could turn into a yak shave. I only added android-studio because android can be built on all three platforms, whereas xcode is almost a prereq for dev on mac platform.